airflow/providers/google/cloud/transfers/gcs_to_bigquery.py

```diff
@@ -319,9 +319,10 @@ def execute(self, context: Context):
                     location=self.location,
                     use_legacy_sql=False,
                 )
-            row = list(bq_hook.get_job(job_id=job_id, location=self.location).result())
-            if row:
-                max_id = row[0] if row[0] else 0
+            result = list(bq_hook.get_job(job_id=job_id, location=self.location).result())
+            if result and len(result) > 0:
+                row = result[0]
```

Comment:
```suggestion
            result = bq_hook.get_job(job_id=job_id, location=self.location).result()
            row = next(iter(result), None)
            if row is not None:
```

This should be more efficient.

Reply:
Thanks! Looks good, and tested fine locally.